plugin/core/views.py

```diff
@@ -837,8 +837,8 @@ def diagnostic_severity(diagnostic: Diagnostic) -> DiagnosticSeverity:
     return diagnostic.get("severity", DiagnosticSeverity.Error)
 
 
-def diagnostic_source(diagnostic: Diagnostic) -> str:
-    return diagnostic.get("source", "unknown-source")
+def diagnostic_source(diagnostic: Diagnostic) -> Optional[str]:
+    return diagnostic.get("source")
```

Comment:
Not really that useful to keep this function now that it doesn't have any logic besides getting dict property.